Cargo.toml

```
@@ -38,3 +38,5 @@ default = ["blocking", "async", "async-https"]
 blocking = ["ureq", "ureq/socks-proxy"]
 async = ["reqwest", "reqwest/socks"]
 async-https = ["async", "reqwest/default-tls"]
+async-https-native = ["async", "reqwest/native-tls"]
+async-https-rustls = ["async", "reqwest/rustls"]
```

I think we're supposed to use the feature `request/rustls-tls` which defaults to using [webpki-roots](https://github.com/rustls/webpki-roots), or do you want manually set your root certs? In that case we'd need `request/rustls-tls-manual-roots`.

---

Or can add both options:
```suggestion
async-https-rustls = ["async", "reqwest/rustls-tls"]
async-https-rustls-manual-roots = ["async", "reqwest/rustls-tls-manual-roots"]
```